evm/deploy-its.js

```diff
@@ -346,36 +348,43 @@ async function deployAll(config, wallet, chain, options) {
             continue;
         }
 
-        if (isHyperliquidChain(chain) && shouldUseBigBlocks(key)) {
+        if (deployment.useHyperliquidBigBlocks) {
             await switchHyperliquidBlockSize(options, true, chain);
         }
 
         printInfo(`Deploying ${deployment.name}`);
 
-        const contract = await deployment.deploy();
-
-        if (key === 'interchainTokenFactoryImplementation') {
-            itsFactoryContractConfig.implementation = contract.address;
-        } else if (key === 'interchainTokenFactory') {
-            itsFactoryContractConfig.address = contract.address;
-        } else {
-            contractConfig[key] = contract.address;
-        }
-
-        if (isHyperliquidChain(chain) && shouldUseBigBlocks(key)) {
-            await switchHyperliquidBlockSize(options, false, chain);
-        }
-
-        printInfo(`Deployed ${deployment.name} at ${contract.address}`);
-
-        saveConfig(config, options.env);
-
-        if (chain.chainId !== 31337) {
-            await sleep(5000);
-        }
-
-        if (!(await isContract(contract.address, provider))) {
-            throw new Error(`Contract ${deployment.name} at ${contract.address} was not deployed on ${chain.name}`);
+        try {
+            const contract = await deployment.deploy();
```

Comment:
only this statement needs to be in the try block as it can fail, not the post deploy cmds